packages/expo-router/src/modal/Modal.tsx

```diff
@@ -139,6 +147,7 @@ export function Modal(props: ModalProps) {
         uniqueId: newId,
         parentNavigationProp: navigation,
         detents: detents ?? 'fitToContents',
```

Comment:
I don't think we should default to 'fitToContents'. This would turn the `formSheet` into a `bottomSheet` whenever `presentationStyle` is set to `formSheet`. @Ubax, what do you think?

Reply:
Hmm, I think that having either `[1]` or `fitContents` makes sense. I feel like the formsheet should display only the content, rather then take full screen and display a lot of empty background. But this is only my take on this.